demo/src/App.vue

```diff
@@ -424,6 +425,8 @@
                 };
                 vm.minDate = "2020-01-01";
             }, 5000);
+            console.log('formatCents', formatCents(234564560, "MYR", false, true, 4));
```

Comment:
I think this is just for debugging? Supposed to be removed?

Reply:
@shonggeok  nope, this is here for coders to know how to use this function. and this console.log here doesnt effect much as this is just a demo page